### ISSUE TITLE
Enhancement: Adjust the rate of the color change

### ISSUE THREAD

**Issue by sunjam** [jancborchardt/lsd#3]
Hi, I'd like to be able to slow down the speed of the color change as an option so it can take much longer and come off quite subtle.

**Comment by jancborchardt** [comment]
Yes good point – I actually wanted to make it slower, so it can actually be used still. :D @Henni suggested to make it faster. ;)

So this is something we can likely fix directly here. @sunjam do you want to give it a go? You simply need to increase the animation duration at https://github.com/jancborchardt/lsd/blob/master/css/trip.css#L48 (currently 2 seconds). If you find a good value, submit a pull request. :)
